.doc/tutorials/tutorial.rst

```diff
@@ -0,0 +1,220 @@
+|        - name: time                      |                                          |
+|          type: boxplot                   |                                          |
+|      options:                            |                                          |
+|        terminal: QT                      |                                          |
+|                                          |                                          |
++------------------------------------------+------------------------------------------+
+
+
+We now want to compare the collision detector FCL against Bullet with the same parameters as above. This will double the previous boxplots because we are adding another collision detector and thus plot 12 boxplots (RRTstar is still filtered**. If this is too much information you can still filter out additionnal planners. Let's use the ``legends** parameter to bring out the comparision even more.
+
+.. |plot3| image:: planners_cd.svg
+
+
++------------------------------------+------------------------------------+
+| .. code-block:: yaml               |              |plot3|               |
+|                                    |                                    |
+|    gnuplot_config:                 |                                    |
+|      plots:                        |                                    |
+|      - filters:                    |                                    |
+|        ...                         |                                    |
+|        legends:                    |                                    |
+|          - query/collision_detector|                                    |
+|        labels:                     |                                    |
+|        - query/planner             |                                    |
+|        metrics:                    |                                    |
+|        - name: time                |                                    |
+|          type: boxplot             |                                    |
+|      options:                      |                                    |
+|        terminal: QT                |                                    |
+|                                    |                                    |
++------------------------------------+------------------------------------+
+
+*You can swap the legends and labels params to have planners as legends and the collision detectors as labels*
+
+
+To get the full picture, it is often necessary to display several metrics as subplots. If your want to know which of the planners is the fastest in a given scenario, well it's a good idea to compare it against the solved metric, because a planner can be really quick at not solving a goal! Sadly, GNUPlot is not able to automatically resize a multiplot. However, the terminal size is configurable at runtime. You can also use the SVG terminal to somewhat bypass the resize limitation.
```

Comment:
```suggestion
To get the full picture, it is often necessary to display several metrics as subplots. If you want to know which of the planners is the fastest in a given scenario, well it's a good idea to compare it against the solved metric, because a planner can be really quick at not solving a goal! Sadly, GNUPlot is not able to automatically resize a multiplot. However, the terminal size is configurable at runtime. You can also use the SVG terminal to somewhat bypass the resize limitation.
```